### ISSUE TITLE
Update 2.5.0 to 3.0.1;  not working

### ISSUE THREAD

**Issue by mrieglhofer** [iobroker-community-adapters/ioBroker.hue#92]
Hi
I have updated HUE 2.5.0 to 3.0.1 and get the following error messages:

```
hue.0	2020-01-11 16:08:42.787	error	(2930) Uhandeld Rejection TypeError: Cannot read property 'LightState' of undefined at [object Promise]
hue.0	2020-01-11 16:08:42.011	error	(2930) Uhandeld Rejection TypeError: Cannot read property 'LightState' of undefined at [object Promise]
hue.0	2020-01-11 16:08:41.795	error	(2930) Uhandeld Rejection TypeError: Cannot read property 'LightState' of undefined at [object Promise]
hue.0	2020-01-11 16:08:41.517	error	(2930) Uhandeld Rejection TypeError: Cannot read property 'LightState' of undefined at [object Promise]
```
When i change a state in Iobroker, no reaction of the plugs. With HUE App it is working.
I went back to 2.5.0 and it is working again. I tested this 3 times.

Node.js v10.18.1 (same issue with 10.18.0)
NPM  v6.13.4
Hostname  IoBrokerPi4!
Hue Extended works too.
thanks 
Markus

**Comment by foxriver76** [comment]
I need some more information, you talk about plugs - so does it only affect smart plugs or also original bulbs? Also you could try to create new credentials via the adapter, it can be that the changes in the 3rd party lib broke something and you need to create a new user. 

If this doesn't help please show the debug log of adapter startage + some state changes which lead to the error.

**Comment by mrieglhofer** [comment]
I habe only Osram Plugs installed.
I tried to get new credentials, no change. 
In the normal log are only these entries, no error at start.

I get the status in iobroker, but it seems, that commands don’t get back to the bridge.

I first stay at 2.5.0 which works. When i have more time, I try an new update and look, whether I can find something more in the log. 

Thanks for your response.

Markus Rieglhofer

**Comment by foxriver76** [comment]
Can confirm that. I grabbed an old Osram plug out of my boxroom, getting the same error.

**Comment by foxriver76** [comment]
fixed in 3.0.2

**Comment by mrieglhofer** [comment]
Fine. Thank you for your help and your work.
Markus
